.circleci/config.yml

```diff
@@ -179,32 +201,7 @@ jobs:
           path: test-reports
           destination: test-reports
 
-
-  commit:
-    docker:
-      - image: circleci/python:3.8.0
-
-    working_directory: ~/repo
-
-    steps:
-      - checkout
-
-      - add_ssh_keys:
-          fingerprints:
-            - "8c:68:29:28:04:80:38:31:c0:59:d9:3d:65:3e:b7:8c"
-
-      - run:
-          name: push to master
-          command: |
-            git push origin HEAD:master
-
-
-  build-py3.6:
-    docker:
-      - image: gethue/hue:latest-py2 # Should be circleci/python:3.6 at some point
-
-    working_directory: ~/repo
-
+  build-py-three-six-common:
```

Comment:
CircleCI 2.1 does not allow using digits in the jobs' names, thus `py-3.6` becomes `py-three-six`